packages/prisma/migrations/20241127102756_remove_fields_from_evenet_type_translation/migration.sql

```diff
@@ -0,0 +1,33 @@
+/*
+  Warnings:
+
+  - The primary key for the `EventTypeTranslation` table will be changed. If it partially fails, the table could be left without primary key constraint.
+  - You are about to drop the column `id` on the `EventTypeTranslation` table. All the data in the column will be lost.
+  - You are about to drop the column `sourceLang` on the `EventTypeTranslation` table. All the data in the column will be lost.
+  - You are about to drop the column `targetLang` on the `EventTypeTranslation` table. All the data in the column will be lost.
+  - Made the column `uid` on table `EventTypeTranslation` required. This step will fail if there are existing NULL values in that column.
+
+*/
+
+-- Copy data
+UPDATE "EventTypeTranslation" SET 
+  "sourceLocale" = "sourceLang",
+  "targetLocale" = "targetLang";
+
+-- Copy data
+UPDATE "EventTypeTranslation" SET 
+  "uid" = "id";
```

Comment:
@keithwillcode 
I manually added these for the new EventTypeTranslation data that might have been created. We don't need this if we run all migration scripts for every new deployment (but I assumed only the most recent migration.sql runs, am I right?)

Reply:
Why do we need the copying? The copy happened upon release yesterday and we are inserting into both set ms of columns at the moment via code, right?

Reply:
> we are inserting into both set ms of columns at the moment via code, right?

True for `sourceLocale` and `targetLocale` but not for `uid` actually. I will remove the copying for sourceLocale and targetLocale